PURL-TYPES.rst

```diff
@@ -221,6 +207,34 @@ nuget
 
       pkg:nuget/EnterpriseLibrary.Common@6.0.1304
 
+oci-artifact
```

Comment:
If container images are in scope for this, then I would consider renaming this purl type to just `oci`. Not all images conform to [OCI Artifacts](https://github.com/opencontainers/artifacts); e.g. multi-platform images don't fit into that model currently.

Reply:
See [above](https://github.com/package-url/purl-spec/pull/123#discussion_r710606060) where we agreed this would be oci-artifact as images are one of many types of artifacts.

Reply:
naming is as hard as scoping :-) I agree with @jonjohnsonjr if the scope was docker vs oci images we should name it oci or oci-image.

I think longer term though we should probably just scope this as all oci-artifact types stored in one of these oci compatible registries.  I think that's best for the OCI consumer?  borrowing the definitions from the distribution spec:
```
- **Registry**: a service that handles the required APIs defined in this specification
- **Client**: a tool that communicates with Registries
- **Push**: the act of uploading Blobs and Manifests to a Registry
- **Pull**: the act of downloading Blobs and Manifests from a Registry
- **Blob**: the binary form of content that is stored by a Registry, addressable by a Digest
- **Manifest**: a JSON document which defines an Artifact. Manifests are defined under the OCI Image Spec <sup>[apdx-2](#appendix)</sup>
- **Config**: a blob referenced in the Manifest which contains Artifact metadata. Config is defined under the OCI Image Spec <sup>[apdx-4](#appendix)</sup>
- **Artifact**: one conceptual piece of content stored as Blobs with an accompanying Manifest containing a Config
- **Digest**: a unique identifier created from a cryptographic hash of a Blob's content. Digests are defined under the OCI Image Spec <sup>[apdx-3](#appendix)</sup>
- **Tag**: a custom, human-readable Manifest identifier
```
in that context(scope) .. oci-artifacts would mean conceptual pieces of content stored as OCI Blobs with an accompanying OCI Manifest containing a OCI Config